src/main/java/com/datalogics/pdf/samples/forms/FillForm.java

```diff
@@ -177,8 +177,6 @@ public static void fillAcroformFdf(final PDFDocument pdfDocument, final URL inpu
         final FDFService fdfService = new FDFService(pdfDocument);
         fdfService.importForm(fdfDocument);
 
-        // Run calculations on the AcroForm...
-        FormFieldService.getAcroFormFieldManager(pdfDocument).runCalculateScripts();
```

Comment:
Fortunately, Acrobat messes this up, too. If you import the XFDF data and save the file, the barcodes are wrong.